app/controllers/catalog_controller.rb

```diff
@@ -278,8 +276,7 @@ def bulk_jobs_delete
   private
 
   def show_aspect
-    pid = params[:id].include?('druid') ? params[:id] : "druid:#{params[:id]}"
-    @obj ||= Dor.find(pid)
+    @obj ||= find_druid(params[:id])
     @response, @document = get_solr_response_for_doc_id pid
   end
```

Comment:
Several places where `ApplicationController#find_druid` can be called to validate `params[:id]`.